### ISSUE TITLE
Custom icons support

### ISSUE THREAD

**Issue by NikunjKhangwal** [1fexd/LinkSheet#303]
### Feature description

This is probably a very niche request but it would be nice to have if possible. I'm using 3rd party icons in my phone but they look very odd in LinkSheet sharing window. Having the option to use 3rd party icons OR at least the option to choose a shape for icons would make it better. Thanks

![IMG_20240124_114447](https://github.com/1fexd/LinkSheet/assets/99175083/595b67b6-381e-4e8d-acf4-c5e4aa001dfc)
![IMG_20240124_114633](https://github.com/1fexd/LinkSheet/assets/99175083/c79d0414-c2e2-4719-a324-db96d352303f)

**Comment by 1fexd** [comment]
What launcher are you using?

**Comment by NikunjKhangwal** [comment]
Niagara Launcher. The icons are made using icon pack studio

**Comment by NikunjKhangwal** [comment]
Sorry for the off-topic question, Is there any plan for the next release of LinkSheet stable?

**Comment by 1fexd** [comment]
Yes, checkout [Milestone 1](https://github.com/1fexd/LinkSheet/milestone/1). Can't really say when though.
